### ISSUE TITLE
certbot-auto produces a stray newline in the standard output

### ISSUE THREAD

**Issue by melmothx** [certbot/certbot#7012]
Running certbot-auto on Debian Jessie, since the automatic upgrade to 0.34.0 each calls to certbot-auto outputs a new line

```
/usr/local/sbin/certbot-auto -q renew | hexdump -C
00000000  0a                                                |.|
00000001
```

The problem is that this in turn generates an empty cronjob mail. This was not happening with previous versions.

Thanks

**Comment by bmw** [comment]
Thanks for reporting this. We'll be putting out a point release to fix the problem, but you can work around it by adding `--no-permissions-check` to the command line in the meantime.

**Comment by melmothx** [comment]
@bmw thanks!

**Comment by bmw** [comment]
We're currently planning on fixing this in a point release on Monday.

As an aside, if anyone hitting this issue isn't running on Debian 8 or a RHEL 6 based distro, I recommend you take this opportunity to visit https://certbot.eff.org where you can get up to date instructions on how to install Certbot. The issue reported here is `certbot-auto` specific and unless you're on a distro I mentioned above, there are likely better ways for you to install a recent version of Certbot.

**Comment by jsoref** [comment]
@bmw:
```
certbot:
  Installed: (none)
  Candidate: 0.28.0-1~deb9u2
  Version table:
     0.28.0-1~deb9u2 500
        500 http://ftp.us.debian.org/debian stable/main amd64 Packages
     0.28.0-1~bpo9+1 100
        100 http://deb.debian.org/debian stretch-backports/main amd64 Packages
```

```
certbot-auto --no-self-upgrade --version

certbot 0.34.0
```

I'm not sure that something from 2018 counts as current. Also, the fact that the backports repo doesn't actually yield a current binary over the one from debian itself isn't particularly reassuring...

**Comment by bmw** [comment]
The version in Debian being the same one that's in backports has more to due with [Certbot in Debian being updated recently](https://lists.debian.org/debian-stable-announce/2019/01/msg00002.html) than backports not seeing updates.

Also, now that the [ACME protocol has been finalized](https://letsencrypt.org/2019/03/11/acme-protocol-ietf-standard.html), frequent Certbot updates are less important for most users, but one of the core team's main projects right now is a new way to distribute Certbot outside of traditional packaging. `certbot-auto` is a disproportionate cause of issues and we think we can do better. Expect to hear updates about that in the next couple months.

With all of that said, I agree `stretch-backports` is in need of an update. Can you help us with that @hlieberman?

**Comment by adferrand** [comment]
Reopening to make the issue visible and avoid duplicates. This issue is fixed in Certbot 0.34.1 that will be released on next Monday.

I will close this issue once 0.34.1 is released.

**Comment by bmw** [comment]
This was fixed in the 0.34.1 release.
